src/main/java/info/tehnut/soulshards/core/EventHandler.java

```diff
@@ -2,24 +2,20 @@
 
 import info.tehnut.soulshards.SoulShards;
 import info.tehnut.soulshards.api.BindingEvent;
-import info.tehnut.soulshards.api.IBinding;
 import info.tehnut.soulshards.api.ISoulWeapon;
+import info.tehnut.soulshards.core.config.ConfigSoulShards;
 import info.tehnut.soulshards.core.data.Binding;
 import info.tehnut.soulshards.core.data.MultiblockPattern;
 import info.tehnut.soulshards.core.data.Tier;
 import info.tehnut.soulshards.item.ItemSoulShard;
-import net.fabricmc.fabric.events.PlayerInteractionEvent;
+import net.fabricmc.fabric.api.event.player.UseBlockCallback;
 import net.minecraft.block.BlockState;
 import net.minecraft.enchantment.EnchantmentHelper;
 import net.minecraft.entity.LivingEntity;
 import net.minecraft.entity.damage.DamageSource;
 import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.item.ItemStack;
-import net.minecraft.sortme.ItemScatterer;
-import net.minecraft.util.ActionResult;
-import net.minecraft.util.Hand;
-import net.minecraft.util.Identifier;
-import net.minecraft.util.TypedActionResult;
+import net.minecraft.util.*;
```

Comment:
A wildcard import for this is overkill

Reply:
Blame IDEA, which auto-collapsed it after 5 imports. Fixed.